app/src/main/java/com/sdpteam/connectout/map/MapModel.java

```diff
@@ -4,32 +4,9 @@
 
 import com.sdpteam.connectout.event.Event;
 
-import java.util.ArrayList;
 import java.util.List;
 
-public class MapModel implements InterfaceMapModel {
-    private ArrayList<Event> dataSet = new ArrayList<>(); //To be defined at a later stage
+public interface MapModel {
```

Comment:
I believe that the way the MVVM of the map was refactored is clearly more intuitive.
However, I would argue that interfaces names should end with the "Manager" word in order to respect naming convention in comparaison with `EventDataManager` and `ProfileDataManager`

Reply:
For me MVVM = Model, View, ViewModel
There is no ModelManager in here. I would rather change the other you mentioned, but anyway ok if you want